### DISCUSSION TITLE
Instal Pip

### DISCUSSION THREAD

**Discussion by webdsf** [mediacms-io/mediacms#34] [Q&A]
Hello. I would like to install additional packs from Pip. How can this be done to make everything work? For example, to expand the place, there are packs that could do this.

**Comment by mgogoulos** [comment]
hello, what are you trying to achieve? It's better if you can use the Discussions section, please close this and open a discussion in https://github.com/mediacms-io/mediacms/discussions!

**Reply by mgogoulos** [reply]
Just saw that Github allows to move issues to discussion and did it myself ;)
